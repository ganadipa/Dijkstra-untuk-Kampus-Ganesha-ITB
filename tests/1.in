4
1
2
42
33
