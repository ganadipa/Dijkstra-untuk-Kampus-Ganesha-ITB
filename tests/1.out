2 75
17 229
2 0
17 154
3 0
17 70
4 0
17 90
